### ISSUE TITLE
Does not work on windows 10.

### ISSUE THREAD

**Issue by HoDANG** [IdolKeg/AnimaPaper#16]
The wallpaper does not change. Only a popup window appears. 
![wallpaper](https://cloud.githubusercontent.com/assets/15304194/26386669/18d2e6ee-4016-11e7-9611-b9b84c6a2328.gif)
The **Windows 7 and below** option has no effect either.

**Comment by IdolKeg** [comment]
Check if you have windows media player installed in windows features since it uses it.

**Comment by HoDANG** [comment]
It is already installed. I tried uninstalling and reinstalling it, but the issue still persists.

**Comment by IdolKeg** [comment]
That's an odd bug. Can you give me more details about your system please ?

**Comment by HoDANG** [comment]
Yes. The desktop is running Windows 10 on the Creators update. I swap between high contrast mode, and the default windows 10 theme, but neither theme allows AnimaPaper to function properly. 
This is my DxDiag: [https://pastebin.com/raw/WtJYk69U](https://pastebin.com/raw/WtJYk69U)
This is my System Info 32: [https://pastebin.com/raw/g882PUyb](https://pastebin.com/raw/g882PUyb) 
If there is any other information you need. please ask.

**Comment by IdolKeg** [comment]
Do you have codecs pack installed ? or filters pack ? Since I guess people are not using WMP but another player to watch stuff they may have installed a bunch of other things. 
Maybe, there is a chance that there is a conflict if you have filter / codec pack with WMP.

**Comment by HoDANG** [comment]
No I do not have any codec packs installed. I use a portable version of VLC to play video files, and a portable version of Foobar2000 to play audio files.
